pkg/runner/client.go

```diff
@@ -151,7 +151,7 @@ func CreateClient(cfg *Config, promMetrics *api.Metrics) (*pgclient.Client, erro
 	}
 
 	if elector == nil && !cfg.APICfg.ReadOnly {
-		log.Warn(
+		log.Debug(
 			"msg",
 			"No adapter leader election. Group lock id is not set. "+
 				"Possible duplicate write load if running multiple connectors",
```

Comment:
Let's change the if to `if (elector == nil && !cfg.APICfg.HighAvailability) && !cfg.APICfg.ReadOnly`
```suggestion
		log.Info(
			"msg",
			"Prometheus HA is not enabled",
```